lib/register.js

```diff
@@ -7,11 +7,22 @@ module.exports = handler;
 function handler({logger}) {
   return (req, res) => {
     logger.info(`received ${req.method} from ${req.protocol}/${req.source_address}:${req.source_port}`);
-    logger.info(req.registration, 'registration details');
+	
+    const fromUri = req.getParsedHeader('From').uri;
+    const aor = fromUri.match(/^sip:(.*)$/)[1];
+
+    req.registration = {
+	  aor: aor,
+	  expires: req.msg.headers.expires,
```

Comment:
Expires value could be in the Expires header, but it could also be in the Contact header.  That's one reason I think it would be better to let the regParser middleware handle some of the  basic parsing of a registration request.  The regParser middleware should come before the auth middleware